2.6/root/usr/share/container-scripts/mongodb/init-replset.sh

```diff
@@ -43,7 +43,7 @@ function initiate() {
 
   info "Creating MongoDB users ..."
   mongo_create_admin
-  mongo_create_user "-u admin -p ${MONGODB_ADMIN_PASSWORD}"
+  [[ -v CREATE_USER ]] && mongo_create_user "-u admin -p ${MONGODB_ADMIN_PASSWORD}"
```

Comment:
`mongo_create_admin` line creates admin user.

`mongo_create_user` creates "normal" user (passed string contains parameters for `mongo` shell - needs to know how to login)